redisCache.py

```
@@ -41,16 +45,28 @@
-        print("\033[94mINFO-DB:\033[0m  \033[92mRedis connected successfully!\033[0m")
-    except Exception as e:
-        print(f"\033[91mERROR-DB:\033[0m Redis connection failed: {e}")
+    retries = 0
+    max_retries = 100
+    retry_delay = 12  # seconds
+
+    while retries < max_retries:
+        try:
+            await redis.ping()
+            print("\033[94mINFO-DB:\033[0m  \033[92mRedis connected successfully!\033[0m")
+            return
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)